src/Components/Facility/FacilityHome.tsx

```diff
@@ -475,15 +475,21 @@ export const FacilityHome = (props: any) => {
 const LiveMonitoringButton = () => {
   const facilityId = useSlug("facility");
   const [location, setLocation] = useState<string>();
+  const authUser = useAuthUser();
+
+  const userRoles = ["StateAdmin", "DistrictAdmin", "Doctor"];
```

Comment:
Just a small suggestion to use more descriptive variable names in future. Here `userRoles` does not convey the purpose as to why we have these 3 roles here in a list. Something like `allowedUserRoles` or `permittedRoles` would make more sense. Also seems like in most places we refer to these as user *types* and not *roles*. 

No issues, just small scopes of improvements!

Reply:
done
can u check once @Ashesh3

Reply:
@AshrafMd-1 Great work! Looks good now.